arangosh/Import/ImportFeature.cpp

```diff
@@ -258,6 +265,24 @@ void ImportFeature::validateOptions(std::shared_ptr<options::ProgramOptions> opt
       FATAL_ERROR_EXIT();
     }
   }
+  for (auto const& it : _datatypes) {
+    auto parts = StringUtils::split(it, '=');
+    if (parts.size() != 2) {
+      TRI_ASSERT(parts.size() == 1);
```

Comment:
should this be addressed as well in lines 465 and 466, where it's verified again if there's an assignment expression within the string?

Reply:
I mean, if there's no assignment, to assert `parts.size() == 1`